lib/widget/button.dart

```diff
@@ -121,7 +122,10 @@ class ButtonState extends WidgetState<Button> {
             color: widget._controller.borderColor ?? defaultShape.side.color,
             width: widget._controller.borderWidth?.toDouble() ?? defaultShape.side.width);
       }
-
+      widget._controller.borderGradient!=null?
```

Comment:
why use the shorthand notation on its own? It's harder to read. Either use proper if/else or do short-hand on the assignment